aten/src/ATen/core/TensorMethods.h

```diff
@@ -1036,31 +1036,31 @@ inline Tensor & Tensor::addcmul_(const Tensor & tensor1, const Tensor & tensor2,
 inline Tensor & Tensor::addcdiv_(const Tensor & tensor1, const Tensor & tensor2, Scalar value) {
     return type().addcdiv_(*this, tensor1, tensor2, value);
 }
-inline Tensor & Tensor::random_(int64_t from, int64_t to, Generator * generator) {
+inline Tensor & Tensor::random_(int64_t from, int64_t to, Generator* generator) {
     return type().random_(*this, from, to, generator);
 }
-inline Tensor & Tensor::random_(int64_t to, Generator * generator) {
+inline Tensor & Tensor::random_(int64_t to, Generator* generator) {
     return type().random_(*this, to, generator);
 }
-inline Tensor & Tensor::random_(Generator * generator) {
+inline Tensor & Tensor::random_(Generator* generator) {
     return type().random_(*this, generator);
 }
-inline Tensor & Tensor::uniform_(double from, double to, Generator * generator) {
+inline Tensor & Tensor::uniform_(double from, double to, Generator* generator) {
     return type().uniform_(*this, from, to, generator);
 }
-inline Tensor & Tensor::normal_(double mean, double std, Generator * generator) {
+inline Tensor & Tensor::normal_(double mean, double std, Generator* generator) {
     return type().normal_(*this, mean, std, generator);
 }
-inline Tensor & Tensor::cauchy_(double median, double sigma, Generator * generator) {
```

Comment:
This extra space shouldn't show up. I know it might be tedious to resolve this, but it's an unecessary merge conflict in the making.